plc4j/drivers/c-bus/src/main/java/org/apache/plc4x/java/cbus/configuration/CBusTcpTransportConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.plc4x.java.cbus.configuration;
+
+import org.apache.plc4x.java.transport.tcp.DefaultTcpTransportConfiguration;
+
+public class CBusTcpTransportConfiguration extends DefaultTcpTransportConfiguration {
+
+    @Override
+    public int getDefaultPort() {
+        return 123;
```

Comment:
One thing noticing here. Guess this is a placeholder but there is a constant in the mspec for that

Reply:
or should be....

Reply:
Well ... before refactoring it seems someone had "123" before ... I just took that over.

Reply:
fixed it